src/main/webapp/scripts/forum-script.js

```diff
@@ -179,3 +187,37 @@ function collapseReplies(idHandler) {
   $('#' + elementId + ' .expand-button').css('display', 'inline-block');
   $('#' + elementId + ' .collapse-button').css('display', 'none');
 }
+
+/**
+ *  Onclick handler for a user searching to reload forum with search parameter
+ */
+function search() {
+  const search = $('#search-input').val();
+  $('#search-input').val('');
+  const id = -1;
+  const placeholder = $('#forum-placeholder');
+  expandForum(placeholder, id, search);
+}
+
+/**
+ *  Returns whether a word of the search is contained with in the text
+ *
+ *  @param {String} text contents of a question
+ *  @param {String} search contents of the user search
+ *
+ *  @return {Boolean} whether there is a word of the search within the text
+ */
+function containsSearch(text, search) {
+  const stopWords = ['a', 'to', 'and', 'how', 'the', 'when', 'what', 'why',
```

Comment:
what is a 'stopWord'

Reply:
just for my own understanding

Reply:
A stop word is a commonly used word that search engines often ignore https://en.wikipedia.org/wiki/Stop_words